integration_test/testdata/konfig_localhost.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
```

Comment:
why do we need this file?
As far as I can tell you don't need to fetch the plugin from a hardcoded port number.
All you care about while implementing `--manifest-url` is that manifest is read from URL and then you can pass `--archive` override.

you put `uri: http://ignored`

Reply:
I added a testcase in which `--archive` is not specified along with `--manifest-url`. How can we ignore the URL in that case? We would need a valid URL to download the plugin.

Reply:
Done